modules/tensor_mechanics/src/actions/DynamicTensorMechanicsAction.C

```diff
@@ -37,16 +51,120 @@ DynamicTensorMechanicsAction::validParams()
                         "equilibrium under gravity by running a "
                         "quasi-static analysis (by solving Ku = F) "
                         "in the first time step.");
-  params.addParam<bool>(
-      "use_displaced_mesh", false, "Whether to use displaced mesh in the kernels");
+
+  params.addParam<std::vector<AuxVariableName>>(
+      "velocities",
+      std::vector<AuxVariableName>({"vel_x", "vel_y", "vel_z"}),
+      "Names of the velocity variables");
+  params.addParam<std::vector<AuxVariableName>>(
+      "accelerations",
+      std::vector<AuxVariableName>({"accel_x", "accel_y", "accel_z"}),
+      "Names of the acceleration variables");
+
+  params.addParam<Real>("beta", 0.25, "beta parameter for Newmark Time integration");
+  params.addParam<Real>("gamma", 0.5, "gamma parameter for Newmark Time integration");
```

Comment:
We could have more verbose names for the action input. `beta` and `gamma` could be `newmark_beta` and `newmark_gamma` or something of that sort.

Reply:
If you add damping, it may be good that those action input parameters be more descriptive too.

Reply:
How about I make the doc strings a lot more descriptive, and add the
 parameters into a "Newmark" parameter group instead?

Reply:
That makes sense. One option would be to call the group `HHT` and add `alpha` to the group (in addition to `beta` and `gamma`).